File: Ambulance.BLL/Models/PersonModels/PersonExtModel.cs
using Ambulance.Core.Entities;

namespace Ambulance.BLL.Models.PersonModels;

public class PersonExtModel : PersonBaseModel
{
    public int PersonId { get; set; }
    public string Login { get; set; } = null!;
    public UserRoleViewModel Role { get; set; } = null!;
}

File: src/api/models/PersonModels/PersonExtModel.js
import { PersonBaseModel } from "./PersonBaseModel.js";

export class PersonExtModel extends PersonBaseModel {
    constructor({ personId = 0, login = "", role = "", ...baseProps } = {}) {
        super(baseProps);
        this.personId = personId;
        this.login = login;
        this.role = role;
    }
}